boards/librem_l1um_v2/librem_l1um_v2.config

```diff
@@ -1,3 +1,4 @@
+# WARNING: This system remains perpetually vulnerable to Spectre v2 (CVE-2017-5715). Mitigations and microcode updates previously applied are now known to be ineffective due to QSB-107 and related CVEs. If Spectre v2 is a concern in your threat model, consider migrating to a platform with ongoing microcode support.
```

Comment:
L1UM v2 is 9th gen, which [has a microcode update from Intel](https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/blob/main/releasenote.md#microcode-20250512).

Reply:
fixed. Please confirm